cmd/ci-operator-checkconfig/main.go

```diff
@@ -27,19 +28,32 @@ type promotedTag struct {
 	metadata *api.Metadata
 }
 
+type ClusterProfiles []ClusterProfileDetails
```

Comment:
This struct would probably not be defined here, but including it here for now

Reply:
The validation code will need `ClusterProfileDetails`, so that will have to go to `pkg/api/types.go`.